tests/unit/tf/inputs/test_tabular.py

```diff
@@ -220,12 +226,9 @@ def test_tabular_seq_features_avg_embeddings_with_mapvalues(sequence_testing_dat
 def test_embedding_tables_from_schema_infer_dims(sequence_testing_data: Dataset, aggregation: str):
     cat_schema = sequence_testing_data.schema.select_by_tag(Tags.CATEGORICAL)
     embeddings_block = ml.Embeddings(
-        cat_schema,
+        cat_schema.select_by_tag(Tags.CATEGORICAL),
         embedding_dims={"item_id_seq": 15, "test_user_id": 21},
-        infer_embedding_sizes=True,
-        infer_embedding_sizes_multiplier=2.0,
-        infer_embeddings_ensure_dim_multiple_of_8=True,
-        embeddings_initializer="truncated_normal",
```

Comment:
If I'm reading this correctly. This was working as before because **kwargs are passed down to `EmbeddingTable`. 

It looks like this should now fail if passed and will complain about multiple params with the same name passed to `EmbeddingTable`.

Are there any params to **kwargs would be useful to pass to EmbeddingTable now or can that be removed?